test/e2e/main.go

```diff
@@ -55,23 +63,40 @@ func main() {
 }
 
 func runSuite() error {
-	if err := ExpectOperatorAvailable(); err != nil {
-		glog.Errorf("FAIL: ExpectOperatorAvailable: %v", err)
-		return err
-	}
-	glog.Info("PASS: ExpectOperatorAvailable")
 
-	if err := CreateClusterAutoscaler(); err != nil {
-		glog.Errorf("FAIL: CreateClusterAutoscaler: %v", err)
-		return err
+	expectations := []struct {
+		expect func() error
+		name   string
+	}{
+		{
+			expect: ExpectOperatorAvailable,
+			name:   "[k8s][openshift] Expect operator to be available",
+		},
+		{
+			expect: CreateClusterAutoscaler,
```

Comment:
this should be consistent with the Expect* convention

Reply:
I just copy pasted what was already in the job. I may change the name if it's needed though we need to be careful which expectations are valid. At this point it's just about naming though it would make more sense to do `ExpectClusterAutoscalerToBeCreated`. In general "`Expect<Object>ToBe<State>`" or "`Expect<Object>To<Operator>`".